docs/tutorials/user/credit-class-project-batch-management.md

```diff
@@ -0,0 +1,468 @@
+```
+
+For more information about the command, add `--help` or check out [the docs](../../commands/regen_tx_ecocredit_update-project-admin.md).
+
+To update project metadata, run the following command:
+
+```sh
+regen tx ecocredit update-project-metadata [project-id] [new-metadata]
+```
+
+For more information about the command, add `--help` or check out [the docs](../../commands/regen_tx_ecocredit_update-project-metadata.md).
+
+## Credit Batch
+
+Now that we have a project, we can create a credit batch and issue credits. Only an account listed as a credit class issuer can create a credit batch for projects within the credit class.
+
+A credit batch is a vintage of credits for a given project and at the time of creation, credits are issued to a list of recipients, which can include accounts representing buffer pools.
+
+For more information about credit batches, see [Ecocredit Concepts](../../modules/ecocredit/01_concepts.md).
+
+### Create Batch
+
+We are now ready to create a credit batch and to use the IRI we generated earlier in this tutorial. The IRI for the credit batch can be used in the next command as the value of `[metadata]`.
+
+To create a batch, run the following command:
+
+```sh
+regen tx ecocredit create-batch [batch-json]
+```
+
+For more information about the command, add `--help` or check out [the docs](../../commands/regen_tx_ecocredit_create-batch.md).
+
+### Update Batch
+
+A credit batch can only be updated if the batch is "open" and only by the issuer. It is not common or recommended to create "open" credit batches
+unless you are operating a bridge.
+
+To update batch metadata, run the following command:
+
+```sh
+regen tx ecocredit update-batch-metadata [batch-denom] [new-metadata]
+```
+
+For more information about the command, add `--help` or check out [the docs](../../commands/regen_tx_ecocredit_update-batch-metadata.md).
+
+## Conclusion
+
+Congratulations! You have now created a credit class, project, and batch with supporting data using the same standards and practices as Regen Network Development.
+
+### Regen Mainnet
+
+Everything you've done here can also be done using [Regen Mainnet](../../ledger/get-started/live-networks.md#regen-mainnet). All you need to do is update the configuration for the `regen` binary to use a different chain ID and node endpoint (you'll also need to own official REGEN tokens). See [Live Networks](../../ledger/get-started/live-networks.md) for configuration instructions.
+
+### Regen Marketplace
+
+You can now view your new credit class, project, and batch using a version of [Regen Marketplace](https://dev.app.regen.network/) connected to Redwood Testnet. You also might notice there is not much information on the pages but you have some new abilities. Check out [Regen Network Guidebook](https://guides.regen.network/guides/regen-marketplace) to learn about managing credit classes, projects, and batches using the Regen Marketplace application.
```

Comment:
```suggestion
You can now view your new credit class, project, and batch using the [Regen Marketplace -Testnet App](https://dev.app.regen.network/) connected to Redwood Testnet. You also might notice there is not much information on the pages but you have some new abilities. Check out [Regen Network Guidebook](https://guides.regen.network/guides/regen-marketplace) to learn about managing credit classes, projects, and batches using the Regen Marketplace application.
```

Reply:
Not sure I understand what is meant by this: "You also might notice there is not much information on the pages but you have some new abilities." 

Can you clarify what's meant here?

Reply:
Non-anchored metadata is not covered in this tutorial. Users would ideally go to the guides for more information about what they can do in the marketplace application.

Reply:
I still find that phrasing of "you may have some new abilities" kind of confusing. Can you upgrade the wording to be more explicit?

Reply:
Upgraded https://github.com/regen-network/regen-ledger/pull/1965/commits/704fb8153f64f572038d94d3b56b5996a2754bf8